swan-lake/learn-the-platform/ballerina-tooling/ballerina-shell.md

```diff
@@ -1,8 +1,8 @@
 ---
 layout: ballerina-ballerina-shell-left-nav-pages-swanlake
-title: Ballerina Shell
+title: Ballerina Shell [experimental]
 permalink: /learn/ballerina-shell/
-description: The Ballerina Shell is a Read-Evaluate-Print Loop (REPL) for Ballerina.
+description: The Ballerina Shell is an experimental feature that provides a Read-Evaluate-Print Loop (REPL) for Ballerina.
```

Comment:
We need to change the description when the user enters the command `bal shell` as well. It currently prints `Welcome to Ballerina Shell REPL.`.

Reply:
https://github.com/ballerina-platform/ballerina-lang/pull/41042, with this it will be updated